src/functions-and-arrays.js

```diff
@@ -1,18 +1,102 @@
 // Iteration #1: Find the maximum
+function maxOfTwoNumbers(num1, num2){
+  if (num1 > num2) {
+    return num1;
+  }
+  if (num2 > num1) {
+    return num2;
+  }
+  if (num1 === num2) {
+    return num1;
+  }
```

Comment:
You could also use the following as an alternative! 
`if (num1 > num2) {
return num1;
} else {
return num2;
}`